client/src/components/Sidebar/Contacts/Contacts.js

```diff
@@ -7,43 +8,66 @@ import Friends from './Friends';
 import Requests from './Requests';
 import Pending from './Pending';
 
+const useStyles = makeStyles(theme => ({
+  invitationHeadings: {
+    marginBottom: theme.spacing(0)
```

Comment:
```suggestion
    marginBottom: 0
```

Reply:
that should suffice if you want to remove all margins from the bottom

Reply:
Thanks Bonnie. Will fix that